### DISCUSSION TITLE
A shortcut key functionality - Windows

### DISCUSSION THREAD

**Discussion by Prateek64X** [dynobo/normcap#305] [Ideas]
Is there a shortcut key to capture when NormCap is open in system tray. It'd be very convinient.

**Comment by dynobo** [comment]
Hi @Prateek64X,

this is currently not possible. But I really like the idea, so I create a [feature request](https://github.com/dynobo/normcap/issues/306) issue to track it.
Unfortunately, it is not trivial to implement, so I'm not sure when I have time to work on it. But I'm always open to contributions!

Right now, the two options to repeatedly perform captures are:

1. Configure your system to [start NormCap via a Hotkey](https://dynobo.github.io/normcap/#faqs-shortcuts) and _deactivate_ NormCap's "Keep in system tray" setting. Then you can repeatedly capture text via that hotkey.
    - :heavy_plus_sign: Keyboard driven 
    - :heavy_minus_sign: Slower, as startup time of NormCap adds up
    - :heavy_minus_sign: It's not always clear, when NormCap is closed completely and ready for a second capture

2. _Activate_ NormCap's "Keep in system tray" setting. Then you can repeatedly capture texts via right-click on the tray  icon and selecting "Capture".
    - :heavy_plus_sign: Faster, as NormCap is already loaded
    - :heavy_minus_sign: Not accessible via keyboard
    - :heavy_minus_sign: One right-click and one left-click needed
